journalctl: add errors from this boot example (#4601)
tldr-pages/tldr: pages/linux/journalctl.md

`journalctl -b -1`

> - Show all errors from this boot (-p is priority and 3 is for errors):

Reviewer: Is this better?
```suggestion
- Show all messages with priority level 3 (errors) from this boot:
```

> `journalctl -p 3 -b`

Reviewer: We can use long argument format to be easier for understand, and the argument option should be in `{{}}`
```suggestion
`journalctl -b --priority={{3}}`
```
